CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 15.2.0
+
+- Added: `messageArgs` to 76 rules ([#6589](https://github.com/stylelint/stylelint/pull/6589)) ([@kizu](https://github.com/kizu)).
+- Fixed: `annotation-no-unknown` false positives for CSS-in-JS template literals Fixed: `function-name-case` false positives for CSS-in-JS template literals
+  Fixed: `function-no-unknown` false positives for CSS-in-JS template literals
+  Fixed: `unit-no-unknown` false positives for CSS-in-JS template literals
+  Fixed: `value-keyword-case` false positives for CSS-in-JS template literals ([#6666](https://github.com/stylelint/stylelint/pull/6666)) ([@hudochenkov](https://github.com/hudochenkov)).
```

Comment:
It seems Changeset doesn't support multi-line entries...

Reply:
I thought that might be the case. It's okay; they're rare and we can patch up this changelog when we release.